### ISSUE TITLE
Use ordered substitution arguments in `ja` and `ko` translated strings files

### ISSUE THREAD

**Issue by hustoj2** [GetStream/stream-chat-android#3767]
```
Note: For labels, choose the SDK this task applies to and one of the corresponding labels.

After that, remove the rest of the labels, as well as this comment!
```
I can't figure out how to remove the extra labels from this task, so I'm leaving ^ this comment here until that is done.

### Context & proposal

Currently, the translated files for `ja` and `ko` contain some string resources with substitution arguments that aren't properly ordered. Note that the default string resources (and the other translations) for these same strings do have ordered arguments.

This generates build warnings for all clients of this SDK that clutter up reading the build log. See screenshot
<img width="1775" alt="Screen Shot 2022-06-24 at 11 21 16 AM" src="https://user-images.githubusercontent.com/2012846/175577785-fa427ff2-ac89-4463-9a7d-7ffec16dee7a.png">

### Acceptance criteria

To accept this task we need to complete the following items:

- [ ] Builds no longer produce the warnings shown in the attached screenshot

**Comment by JcMinarro** [comment]
Hello @hustoj2 
Thanks for reporting it.
@skydoves could you check it?

**Comment by adasiewiczr** [comment]
Hey @hustoj2,
translations were improved in the PR mentioned above.
Improvement will be shipped in the next release.

**Comment by adasiewiczr** [comment]
hey @hustoj2,
I'm going to close this issue as we just released a new version (5.5.0) that includes the fix mentioned before. You can find more details [here](https://github.com/GetStream/stream-chat-android/releases/tag/v5.5.0).
